docs/vsto/walkthrough-designing-an-outlook-form-region.md

```diff
@@ -155,6 +155,8 @@ ms.workload:
 
 5.  Reopen the **Ann Beebe** contact item.
 
+    In Outlook 365, this can be done in the **Find** group by either opening the Address Book for Contacts or typing Ann Beebe into **Search People**.
```

Comment:
I would remove "365", as that would imply it doesn't work in the desktop version